arch/arm/core/aarch32/mpu/arm_mpu.c

```diff
@@ -350,11 +350,17 @@ int z_arm_mpu_init(void)
 	/* Clean and invalidate data cache if it is enabled and
 	 * that was not already done at boot
 	 */
+#if (CONFIG_CPU_AARCH32_CORTEX_R)
```

Comment:
`if defined(`

Reply:
I updated

Reply:
@duongvunam, seems not fixed with `if defined(`

Reply:
fixed with if defined(